clients/feeder/dbc2val/dbcfeeder.py

```diff
@@ -91,6 +93,8 @@ def getConfig():
 
     cfg['elm.canack'] = elmcfg.getboolean("canack", False)
 
+    cfg['j1939'] = args.j1939
```

Comment:
can we make the convention similar to the other ones, that ist
 - read from config
 - Allow to override from commandline

that would look something like (untested)

```python
    cfg['can.j1939'] = cancfg.getboolean("j1938", False)
    if args.j1939:
        cfg['can.j1939'] = args.j1939
```

I think this would also mean to remove "default" in argparse (otherwise, when true in config, and NO commandline argument, it would still override to `false`